src/test/java/io/managed/services/test/kafka/QuotaKafkaInstanceTest.java

```diff
@@ -0,0 +1,204 @@
+        } catch (Throwable t) {
+            LOGGER.error("failed to clean kafka instance: ", t);
+        }
+
+        try {
+            KafkaMgmtApiUtils.cleanKafkaInstance(noQuotaUserKafkaMgmtApi, KAFKA_INSTANCE_NAME_FAIL);
+        } catch (Throwable t) {
+            LOGGER.error("failed to clean kafka instance: ", t);
+        }
+    }
+
+    @Test(priority = 1)
+    @SneakyThrows
+    public void testQuotaUserFailedCreateDeveloperInstance() {
+        LOGGER.info("Trying to create Developer Kafka instance '{}'", KAFKA_INSTANCE_NAME_FAIL);
+
+        try {
+            var payload = new KafkaRequestPayload()
+                .name(KAFKA_INSTANCE_NAME_FAIL)
+                .cloudProvider(Environment.CLOUD_PROVIDER)
+                .plan(PLAN_DEVELOPER);
+            kafka = KafkaMgmtApiUtils.createKafkaInstance(quotaUserKafkaMgmtApi, payload);
+            fail("Kafka instance creation did NOT fail");
+        } catch (ApiGenericException e) {
+            assertEquals(e.getCode(), 400, "HTTP Status Response");
+            JSONObject jsonResponse = new JSONObject(e.getResponseBody());  
+            assertEquals(jsonResponse.get("code"), KAFKAS_MGMT_21_CODE);
+            assertEquals(jsonResponse.get("reason"), KAFKAS_MGMT_21_REASON + "'" + PLAN_DEVELOPER + "'");
+        }
+    }
+
+    @Test(priority = 2)
+    @SneakyThrows
+    public void testQuotaUserSucceededCreateStandardInstance() {
+        LOGGER.info("Trying to create Standard Kafka instance '{}'", KAFKA_INSTANCE_NAME_QUOTA);
+
+        var payload = new KafkaRequestPayload()
+            .name(KAFKA_INSTANCE_NAME_QUOTA)
+            .cloudProvider(Environment.CLOUD_PROVIDER)
+            .plan(PLAN_STANDARD);
+        kafka = KafkaMgmtApiUtils.createKafkaInstance(quotaUserKafkaMgmtApi, payload);
+        assertEquals(kafka.getName(), KAFKA_INSTANCE_NAME_QUOTA);
+        assertEquals(kafka.getInstanceType() + '.' + kafka.getSizeId(), PLAN_STANDARD);
+    }
+
+    @Test(priority = 3)
+    @SneakyThrows
+    public void testQuotaUserFailedCreateStandardInstanceWhenQuotaIsReached() {
+        LOGGER.info("Trying to create Standard Kafka instance when quota is reached '{}'", KAFKA_INSTANCE_NAME_FAIL);
+
+        try {
+            var payload = new KafkaRequestPayload()
+                .name(KAFKA_INSTANCE_NAME_FAIL)
+                .cloudProvider(Environment.CLOUD_PROVIDER)
+                .plan(PLAN_STANDARD);
+            KafkaMgmtApiUtils.createKafkaInstance(quotaUserKafkaMgmtApi, payload);
+            fail("Kafka instance creation did NOT fail");
+        } catch (ApiForbiddenException e) {
+            assertEquals(e.getCode(), 403, "HTTP Status Response");
+            JSONObject jsonResponse = new JSONObject(e.getResponseBody());  
+            assertEquals(jsonResponse.get("code"), KAFKAS_MGMT_120_CODE);
+            assertEquals(jsonResponse.get("reason"), KAFKAS_MGMT_120_REASON);
+        }
+    }
+
+    @Test(priority = 4)
```

Comment:
this test and  `testNoQuotaUserSucceededCreateDeveloperInstance` don't need to have different priority ( I assume) ? 

I think that with exception of  
`testQuotaUserSucceededCreateStandardInstance`
and 
`testQuotaUserFailedCreateStandardInstanceWhenQuotaIsReached `
we do not  ordering of tests. 

good solution here would be either using following
```
  @Test(priority = 1, dependsOnMethods = {"testNoQuotaUserSucceededCreateDeveloperInstance"})
  testQuotaUserFailedCreateStandardInstanceWhenQuotaIsReached
  ```
  or have no dependency at all (clear instance after it is created in testNoQuotaUserSucceededCreateDeveloperInstance) 
  
and later create two instances in  `testQuotaUserFailedCreateStandardInstanceWhenQuotaIsReached`